src/server/analytic_services/prometheus_reporting.js

```diff
@@ -61,7 +61,13 @@ async function start_server(
     const server = http.createServer(async (req, res) => {
         // Serve all metrics on the root path for system that do have one or more fork running.
         if (fork_enabled) {
-            const metrics = await aggregatorRegistry.clusterMetrics();
+            let metrics;
+            try {
+                metrics = await aggregatorRegistry.clusterMetrics();
+            } catch (err) {
+                dbg.error('start_server: Could not get the metrics, got an error', err);
+                return;
```

Comment:
We probably should emit an event here, Can you add this to our events github issue?
Do you know how we got `operation timed out` on that? what is the suggested solution in case we can't start the metrics server?